tests/Aspire.Workload.Tests/data/nuget8.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!-- Don't use any higher level config files. -->
+  <fallbackPackageFolders>
+    <clear />
+  </fallbackPackageFolders>
+  <packageSources>
+    <clear />
+    <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
+    <add key="built-local" value="%BUILT_NUGETS_PATH%" />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
+    <add key="nuget.org"  value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
I may be wrong, but we may get flagged in compliance for having nuget.org as a feed (even when just used for testing.)

Reply:
Why do we need nuget.org? We should have everything we need in our normal nuget.config (plus the local path)

Reply:
Hm I'll try without that for the tests. I found it useful when dogfooding the sdk+workload.

Reply:
```
ServiceDefaults.csproj : error NU1101:

Unable to find package OpenTelemetry.Exporter.OpenTelemetryProtocol. No packages exist with this id in source(s)
Unable to find package OpenTelemetry.Extensions.Hosting. No packages exist with this id in source(s)
Unable to find package OpenTelemetry.Instrumentation.AspNetCore. No packages exist with this id in source(s)
Unable to find package OpenTelemetry.Instrumentation.Http. No packages exist with this id in source(s)
Unable to find package OpenTelemetry.Instrumentation.Runtime. No packages exist with this id in source(s)

Unable to find package Microsoft.Extensions.Http with version (>= 8.0.0)
Unable to find package Microsoft.Extensions.Configuration.Abstractions with version (>= 8.0.0)
Unable to find package Microsoft.Extensions.DependencyInjection.Abstractions with version (>= 8.0.1)
Unable to find package Microsoft.Extensions.Logging.Abstractions with version (>= 8.0.1)
Unable to find package Microsoft.Extensions.Options with version (>= 8.0.2)
Unable to find package Microsoft.Extensions.Primitives with version (>= 8.0.0)
```

Reply:
```
Unable to find package Microsoft.Extensions.Http with version (>= 8.0.0)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
Unable to find package Microsoft.Extensions.Configuration.Abstractions with version (>= 8.0.0)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
Unable to find package Microsoft.Extensions.DependencyInjection.Abstractions with version (>= 8.0.1)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
Unable to find package Microsoft.Extensions.Logging.Abstractions with version (>= 8.0.1)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
Unable to find package Microsoft.Extensions.Options with version (>= 8.0.2)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
Unable to find package Microsoft.Extensions.Primitives with version (>= 8.0.0)
  - Found 1459 version(s) in dotnet8 [ Nearest version: 8.0.0-rtm.23523.3 ]
  - Found 0 version(s) in /Users/ankj/dev/aspire/artifacts/bin/dotnet-latest/library-packs
```

Reply:
Why don't you have a `<add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />` entry?

Reply:
Fixed.